docs/book/src/cronjob-tutorial/testdata/project/cmd/main.go

```diff
@@ -76,7 +76,8 @@ func main() {
 	var probeAddr string
 	var secureMetrics bool
 	var enableHTTP2 bool
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":0", "The address the metric endpoint binds to. "+
```

Comment:
I think `":0"` means "bind to any open port", whereas `"0"` means "don't start the metrics server", right?

```suggestion
	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metric endpoint binds to. "+
```

Reply:
That works as well. But we can change no problem at all.